docs/platform/porting/system.md

```diff
@@ -2540,3 +2540,353 @@ Example:
   $ sleep 123 &
   $ livedumper -f sleep.core -m $?
 ```
+
+### Stability monitor
+
+The purpose of the stability monitor is to keep track of the usage of system resources with low overhead. The resources that are tracked for maximum and average usage are as follows:
+-  CPU
+-  Memory
+-  Input/Output
+-  Open file descriptors (only maximum value is measured, not average).
+
+Resources are consumed by various processes, classified according to their type as application or service. Applications have the type (native, web app, .NET), and state (foreground, background). The process parameters are distinguished in configuration, and various parameters can be assigned to different services and applications. When excessive use of resources abnormality is detected, an appropriate D-Bus signal is sent, and an optional abnormality report is generated.
```

Comment:
```suggestion 
Resources are consumed by various processes and are classified according to their type: application or service. Applications have the type as native, web app, and .NET, with the state as foreground and background. The process parameters are distinguished in configuration and various parameters can be assigned to different services and applications. When excessive use of resources abnormality is detected, an appropriate D-Bus signal is sent, and an optional abnormality report is generated.
```
Slight modification for better comprehension. Please check for technical accuracy, then only incorporate.